schemas/pom.xml

```diff
@@ -51,8 +51,29 @@
     <module>dublin-core</module>
     <module>iso19110</module>
     <module>iso19139</module>
-    <module>iso19115-3.2018</module>
   </modules>
+
+  <profiles>
+    <profile>
+      <id>schema-iso19115-3-2018</id>
+      <activation>
+        <file><exists>iso19115-3.2018</exists></file>
+      </activation>
+      <modules>
+        <module>iso19115-3.2018</module>
+      </modules>
+    </profile>
+    <profile>
+      <id>hnap</id>
```

Comment:
I though this was going to be removed?  If not then it should be renamed to schema-iso19139.ca.HNAP to be consistent.

Reply:
This is a built-in schema that is handled as a sub module (it was a sub module before i started).

Since it was already here I can use it as a working example of how the metadata101 modules are integrated.

Reply:
This is a built-in schema that is handled as a sub module (it was a sub module before i started).

Since it was already here I can use it as a working example of how the metadata101 modules are integrated.

Reply:
Sorry - I'm confused? 
iso19115-3-2018 is a build in schema that is handled as a submodule.  I see that you have moved    iso19115-3-2018  to a profile that should only include  iso19115-3-2018  module if the folder exists.

HNAP is a submodule as well but it it was not there before you started.  So I don't understand your comment "(it was a sub module before i started)".  I think you  are referring to  iso19115-3-2018 profile but I'm referring to hnap profile.

Reply:
I just noticed in the comment below that you mentioned that you will be adding HNAP and many other profiles in the future.  So it sound like you are not removing HNAP. So in this case I propose that it be renamed to schema-iso19139.ca.HNAP.